### DISCUSSION TITLE
Swapped virtual monitor

### DISCUSSION THREAD

**Discussion by llee8820** [ClassicOldSong/Apollo#152] [Q&A]
Hello, I'm scratching my head trying to solve this. When I run moonlight, it seems the virtual monitor is created correctly. But the game launches on my PC rather than my moonlight client. How do I swap this? Thank you.

**Comment by ClassicOldSong** [comment]
You need to change the virtual dispalay to primary in Windows settings. You only need to do this once.

**Reply by llee8820** [reply]
It seems that the virtual display is set as primary already. The steam windows opens on my Moonlight client, but the game still only shows up on the PC. I also have the "Enforce virtual display primary" option checked

**Reply by ClassicOldSong** [reply]
Then you might need to check your game's configuration. Some games have option to change the display it launches on.

Or you can just disable your physical display and use only the virtual display.

**Reply by llee8820** [reply]
O I think I figured it out. I just manually use the shortcut "win+shift+arrow" to move to virtual monitor and apollo seems to remember that selection the next time. Thanks for the quick responses though!

**Reply by ClassicOldSong** [reply]
You're welcome!
